public/locales/ko-kr.json

```diff
@@ -1418,26 +1440,190 @@
     "Chat API": "채팅 API",
-    "and pick a character.": "그리고 캐릭터를 선택하세요.",
+    "and pick a character.": "그리고 원하는 캐릭터를 선택하세요.",
     "You can browse a list of bundled characters in the": "번들 캐릭터 목록을 검색할 수 있습니다.",
-    "Download Extensions & Assets": "확장 프로그램 및 자산 다운로드",
+    "Download Extensions & Assets": "확장 프로그램 및 에셋 다운로드",
+    "Character Expressions": "캐릭터 감정 표현들",
+    "ext_translate_title": "채팅 번역",
+    "ext_translate_auto_mode": "자동 번역 모드",
+    "ext_translate_mode_none": "없음",
+    "ext_translate_mode_responses": "응답 자동 번역",
+    "ext_translate_mode_inputs": "입력 자동 번역",
+    "ext_translate_mode_both": "응답과 입력 모두 번역",
+    "ext_translate_mode_provider": "번역 공급자",
+    "ext_translate_target_lang": "번역 도착 언어",
+    "ext_translate_clear": "번역 지우기",
+    "ext_sum_title": "채팅 요약",
+    "ext_sum_with": "요약에 사용될 API:",
+    "ext_sum_main_api": "메인 API",
+    "Vector Storage": "벡터 저장소",
     "menu within": "내의 메뉴",
+    "Translate text to English before classification": "분류 전에 텍스트를 영어로 번역합니다.",
+    "Show default images (emojis) if sprite missing": "해당하는 스프라이트가 없으면 기본 이미지 (이모지들)을 표시합니다.",
+    "Image Type - talkinghead (extras)": "이미지 유형 - 토킹 헤드 (부가 사항)",
+    "Classifier API": "분류를 위한 API",
+    "Select the API for classifying expressions.": "감정 이미지들을 분류할 API를 선택하세요.",
+    "Local": "로컬",
+    "Extras": "외부",
+    "LLM": "대형 언어 모델(LLM)",
+    "LLM Prompt": "LLM 프롬프트",
+    "Will be used if the API doesn't support JSON schemas or function calling.": "API가 JSON 스키마나 함수 호출을 지원하지 않는 경우 사용됩니다.",
+    "Default / Fallback Expression": "기본 / 대체 감정들",
+    "Set the default and fallback expression being used when no matching expression is found.": "일치하는 표현이 없을 때 사용되는 기본 및 대체 표현을 설정합니다.",
+    "Custom Expressions": "커스텀 감정들",
+    "Can be set manually or with an _space": "수동으로 설정하거나 또는",
+    "space_ slash command.": "슬래시 명령어로 설정할 수 있습니다.",
+    "Open a chat to see the character expressions.": "캐릭터 감정을 보려면 채팅창을 열어주세요.",
+    "Sprite Folder Override": "스프라이트 폴더 덮어쓰기",
+    "Upload sprite pack (ZIP)": "스프라이트 압축팩 (ZIP) 업로드하기",
+    "Remove all image overrides": "모든 이미지 재정의 제거하기",
+    "Hint:": "힌트:",
+    "Create new folder in the _space": "사용자 데이터 경로에 있는",
+    "folder of your user data directory and name it as the name of the character.": "폴더에 새 폴더를 만들고 해당 폴더의 이름을 캐릭터의 이름으로 지정하세요.",
+    "Put images with expressions there. File names should follow the pattern:": "그리고 그곳에 표정이 담긴 이미지를 넣으세요. 파일 이름은 다음 패턴을 따라야 합니다:",
+    "Sprite set:": "스프라이트 세트:",
+    "sd_free_extend_txt": "무료 모드 프롬프트 확장",
     "Confused or lost?": "혼란스러우시거나 길을 잃으셨나요?",
     "click these icons!": "이 아이콘을 클릭하세요!",
     "in the chat bar": "채팅 바에서",
-    "SillyTavern Documentation Site": "SillyTavern 문서 사이트",
+    "SillyTavern Documentation Site": "실리태번 공식 문서 사이트",
     "Extras Installation Guide": "Extras 설치 안내서",
     "Still have questions?": "아직 질문이 있으신가요?",
-    "Join the SillyTavern Discord": "SillyTavern Discord에 참여하세요",
+    "Join the SillyTavern Discord": "실리태번 디스코드에 참여하세요",
```

Comment:
I'm so sorry. I was busy with other things and just now checked your comments. You can ignore the claude haiku 1031 version; I added that incorrectly.  As for the brand name, Korean users of SillyTavern call it 실리태번 (SillyTavern), so it doesn't really matter, but it's your repository, so if you want to keep the original brand name, that's fine with me.

This repository was originally a fork I made for Korean users, which I periodically updated from yours.  It seems things got complicated because I sent all the PRs to your main branch instead of a new one.  Later, when I have time, I'll clean things up in a new repository and send you a new PR. Thank you for looking at my PR, and I apologize for the confusion.

Reply:
> Shouldn't we keep the brand name intact? 

And you're right, brand names like "horde" should definitely keep their original names.  And thank you for letting me know about the strings that no longer exist. I'll make a note of that and fix them later.